packages/subapp-web/lib/load.js

```diff
@@ -156,11 +156,11 @@ Response: ${err || body}`
               .concat(cdnJsBundles[ep])
               .reduce((a, jsBundle) => {
                 const ext = Path.extname(jsBundle);
-                if (ext === ".js") {
+                if (ext === ".js" && !jsBundle.endsWith(".hot-update.js")) {
                   if (context.user.headEntries) {
                     headSplits.push(`<link rel="preload" href="${jsBundle}" as="script">`);
                   }
-                  a.push(`<script src="${jsBundle}" async></script>`);
+                  a.push(`<script src="${jsBundle}" async crossorigin="anonymous"></script>`);
```

Comment:
Hi @ashuverma Thanks for changes. 
Looks like this changes is causing a test failure for `should load bundles for the subapp` -  ``Error: Timeout of 2000ms exceeded. ``. Do you know what is going wrong ?

Reply:
Any examples where I can see this?